modules/registry-operator-configuration-resource-overview-openstack-swift.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring-registry-storage/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-operator-configuration-resource-overview-openstack-swift_{context}"]
+= Image Registry Operator configuration parameters for OpenStack Swift
+
+The following configuration parameters are available for OpenStack Swift
+registry storage.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`authURL`
+|This value is optional.
```

Comment:
These descriptions should be filled out if we have the info available, as this isn't very helpful atm.

Reply:
I agree. This isn't actually building in 4.2, it's serving as a framework for when the UPI GAs. We will be able to populate the descriptions at that point.